cairo_programs/blake2s_hello_world_hash.cairo

```diff
@@ -0,0 +1,20 @@
+%builtins range_check bitwise
+
+from starkware.cairo.common.alloc import alloc
+from starkware.cairo.common.cairo_blake2s.blake2s import blake2s
+from starkware.cairo.common.cairo_builtins import BitwiseBuiltin
+
+#Computes the hash of "Hello World"
+func main{range_check_ptr, bitwise_ptr : BitwiseBuiltin*}():
+    alloc_locals
+    let inputs: felt* = alloc()
+    assert inputs[0] = 1819043144
+    assert inputs[1] = 1870078063
+    assert inputs[2] = 653255322
```

Comment:
Cairo supports expressing this as short string literals up to 31 characters.

Reply:
Applied change!